src/commands/rover/WhoisCommand.js

```diff
@@ -218,7 +217,6 @@ class WhoisCommand extends Command {
 
         editMessage.edit({ embed: embed }).catch(console.error)
```

Comment:
I would suggest that we remove this otherwise we would be editing the embed twice without much of a good reason.

Reply:
The ScriptingHelpers and devforum have to be edited in after the initial embed as ScriptingHelpers may take a few seconds to respond so this method was suggested by eryn

Reply:
If that's so it would make more sense to push past usernames before scripting helpers info, but it's not much of a concern.